README.md

```diff
@@ -7,11 +7,22 @@
 
 ### _Instagram_, dummified
 
-![Issues](https://img.shields.io/github/issues/narayan954/dummygram?color=brightgreen)
-![Pull requests](https://img.shields.io/github/issues-pr/narayan954/dummygram)
-![Forks](https://img.shields.io/github/forks/narayan954/dummygram)
-![Stars](https://img.shields.io/github/stars/narayan954/dummygram)
-![Licence](https://img.shields.io/github/license/narayan954/dummygram?color=orange)
+<div align="center">
+      <img src="https://forthebadge.com/images/badges/built-with-love.svg" />
+      <img src="https://forthebadge.com/images/badges/uses-brains.svg" />
```

Comment:
it uses react more than brains right?

Reply:
> it uses react more than brains right?

yahh

Reply:
so is "uses-brains" okay or should it be "uses-react". Whatever suits, you may apply